server/socket-handlers/general-socket-handler.js

```diff
@@ -2,31 +2,36 @@ const { log } = require("../../src/util");
 const { Settings } = require("../settings");
 const { sendInfo } = require("../client");
 const { checkLogin } = require("../util-server");
-const GameResolver = require("gamedig/lib/GameResolver");
+const { games } = require("gamedig");
 const { testChrome } = require("../monitor-types/real-browser-monitor-type");
 const fs = require("fs");
 const path = require("path");
 
-let gameResolver = new GameResolver();
-let gameList = null;
-
 /**
  * Get a game list via GameDig
- * @returns {object[]} list of games supported by GameDig
+ * @returns {object} list of games supported by GameDig
  */
 function getGameList() {
-    if (gameList == null) {
-        gameList = gameResolver._readGames().games.sort((a, b) => {
-            if ( a.pretty < b.pretty ) {
-                return -1;
-            }
-            if ( a.pretty > b.pretty ) {
-                return 1;
-            }
-            return 0;
-        });
-    }
-    return gameList;
+    let gamelist = [];
+    gamelist = Object.keys(games).map(key => {
+        const item = games[key];
+        return {
+            keys: [ key ],
+            pretty: item.name,
+            options: item.options,
+            extra: item.extra || {}
+        };
+    });
+    gamelist.sort((a, b) => {
+        if ( a.pretty < b.pretty ) {
+            return -1;
+        }
+        if ( a.pretty > b.pretty ) {
+            return 1;
+        }
+        return 0;
+    });
+    return gamelist;
```

Comment:
Resolves https://github.com/louislam/uptime-kuma/pull/4949/files#r1798938481

```suggestion
    let gameList = [];
    gameList = Object.keys(games).map(key => {
        const item = games[key];
        return {
            keys: [ key ],
            pretty: item.name,
            options: item.options,
            extra: item.extra || {}
        };
    });
    gameList.sort((a, b) => {
        if ( a.pretty < b.pretty ) {
            return -1;
        }
        if ( a.pretty > b.pretty ) {
            return 1;
        }
        return 0;
    });
    return gameList;
```